frontend/src/queries/nodes/WebOverview/WebOverview.tsx

```diff
@@ -136,7 +139,19 @@ const formatPercentage = (x: number, options?: { precise?: boolean }): string =>
     return (x / 100).toLocaleString(undefined, { style: 'percent', maximumFractionDigits: 0 })
 }
 
-const formatSeconds = (x: number): string => humanFriendlyDuration(Math.round(x))
+const formatSeconds = (x: number): string => {
```

Comment:
i wonder if you could use or update https://github.com/PostHog/posthog/blob/12ada6ae869edeed46bcac4b96065c085968193b/frontend/src/lib/utils.tsx#L489 or https://github.com/PostHog/posthog/blob/12ada6ae869edeed46bcac4b96065c085968193b/frontend/src/lib/utils.tsx#L500

Reply:
I was scared to update stuff that's used in a lot of places 😬

Reply:
i've learned so much (and broken then fixed so many things changing these things 🤣)

we should at least keep them together so the future traveller can see what's happening

i've got a PR where I want to figure out forcing decimal places for durations in insights https://github.com/PostHog/posthog/pull/24980 similar thing where I've had to stop to think about it because so many things changed 🤣

Reply:
like i say... not blocking just maybe more neat

Reply:
I'll do this in a follow-up so that it can be reviewed on its own